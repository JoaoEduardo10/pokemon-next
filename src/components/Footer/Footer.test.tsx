import { render } from '@testing-library/react';
import { Footer } from '.';

describe('<Footer />', () => {
	it('should render a footer', () => {
		const { container } = render(<Footer />);

		expect(container).toMatchSnapshot();
	});
});