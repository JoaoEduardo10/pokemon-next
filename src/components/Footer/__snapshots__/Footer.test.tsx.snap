// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render a footer 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      <span>
        PokeNext
      </span>
       Â© 2022
    </p>
  </footer>
</div>
`;
